my journey as a developer (#97)
collab-community/journey-book: journeys/AzeezTM.md

# <h1 align="center">[AzeezTM](https://github.com/AzeezTM)' journey</h1>
<h2 align="center">A front-end developer from Nigeria</h2>

> - i am Mubarak, from Nigeria, i am a front-end developer,a Wildlife Conservationist and an Ecotourist.

Reviewer: ```suggestion
- I am Mubarak, from Nigeria, I am a front-end developer, A Wildlife Conservationist and an Ecotourist.
```

> - i like to tour, i have travelled and visted most of the tourist sites in southwestern Nigeria.

Reviewer: ```suggestion
- I like to tour, I have travelled and visited most of the tourist sites in southwestern Nigeria.
```

> - i am currently learning some programming languages so that i will use it to build projects that will be useful in conserving wildlife and to create innovative travel software solutions.

Reviewer: ```suggestion
- I am currently learning some programming languages so that I will use it to build projects that will be useful in conserving wildlife and to create innovative travel software solutions.
```

## My Journey

> - i started coding this january, 2022 where i learnt how to code at SQI College of ICT,Ibadan, Nigeria.

Reviewer: ```suggestion
- I started coding this January, 2022 where I learnt how to code at SQI College of ICT, Ibadan, Nigeria.
```

> - i started coding this january, 2022 where i learnt how to code at SQI College of ICT,Ibadan, Nigeria.

Reviewer: ```suggestion
- i started coding January, 2022 where i learnt how to code at SQI College of ICT,Ibadan, Nigeria.
```

don't inculde the word this because in future, after 5 years "this" will make no sense.

> - i was taught html,css,bootstrap within two months which i designed some projects within the two months.

Reviewer: ```suggestion
- I was taught HTML, CSS, Bootstrap within two months which i designed some projects within the two months.
```

> - i was taught html,css,bootstrap within two months which i designed some projects within the two months.

Reviewer: Can you include how you learnt, who taught you eg: youtube, friends, etc.

Reviewer: @AzeezTM Please do this.

> - i also learnt javascript for two months which i also work on some projects like Picture puzzle game,Alarm,Calculator, Todo List. I was given a picture puzzle game as my javascript project which i presented.i used drag and drop API to make the picture draggable but i am currently working on the last part,i want a modal to display once the player is able to complete the game.

Reviewer: ```suggestion
- I also learnt JavaScript for two months which I also work on some projects like Picture Puzzle Game, Alarm, Calculator, To-do List. I was given a picture puzzle game as my JavaScript project which I presented. I used drag and drop API to make the picture draggable but I am currently working on the last part, I want a modal to display once the player is able to complete the game.
```

> - i just finish learning Reactjs and i have built a Banking App(https://github.com/AzeezTM/Banking_App) and i'm currently learning Typescript.

Reviewer: ```suggestion
- I just finish learning React-js and I have built a Banking App(https://github.com/AzeezTM/Banking_App) and I'm currently learning Typescript.
```

> - i am currently looking for an internship to get more experience as a Software Developer

Reviewer: ```suggestion
- I am currently looking for an internship to get more experience as a Software Developer
```

## Connect with me

[Github](https://github.com/AzeezTM),

> [Twitter](@Azeeztim)

Reviewer: ```suggestion
[Twitter](https://twitter.com/Azeeztim)
```

hyperlink not stated properly, just added username

Reviewer: @AzeezTM Please do this.

> Facebook: Azez Tunji Mubarak Atm

Reviewer: Can you link the Facebook?

Reviewer: @AzeezTM Please do this.

> Linkedin: Tunji Mubarak Azeez

Reviewer: Can you link the LinkedIn

Reviewer: @AzeezTM Please do this.
